Makefile.example

```diff
@@ -12,7 +12,10 @@ PKG := github.com/drud/repo_name
 # UPSTREAM_REPO ?= full/upstream-docker-repo
 
 # Top-level directories to build
-SRC_DIRS := files drudapi secrets utils
+SRC_DIRS := cmd pkg
+
+# Golang binary name
+BIN_NAME := $$PWD
```

Comment:
The problem with this is the assumption of a single binary - There can be many outputs from a build. How about BINARIES ? I think that will work.